lib/sync/index.js

```diff
@@ -67,7 +67,10 @@ function invoke(dataset_id, params, callback) {
   }
   var fnHandler = module.exports[fn];
 
-  return fnHandler(dataset_id, params, callback);
+  start(function(err) {
+    if (err) return callback(err);
```

Comment:
just to be consistent, can we move the return statement to a newline like everywhere else